lib/ibanity/configuration.ex

```diff
@@ -77,7 +77,7 @@ defmodule Ibanity.Configuration do
     |> URI.parse()
     |> URI.merge("/#{product}")
     |> to_string()
-    |> ApiSchema.fetch(default_app_options(), Mix.env())
+    |> ApiSchema.fetch(default_app_options(), Application.get_env(:ibanity, :test) && :test)
```

Comment:
This would be just `Application.get_env(:ibanity, :env)` if we go along with my suggestion.